game/world/world.cpp

```diff
@@ -117,6 +117,19 @@ void World::insertPlayer(std::unique_ptr<Npc> &&npc, std::string_view waypoint)
     game.script()->setInstanceNPC("HERO",*npcPlayer);
   }
 
+void World::setPlayer(Npc* npc) {
+  if(npc==nullptr)
+    return;
+  npcPlayer->setProcessPolicy(Npc::ProcessPolicy::AiNormal);
+  npcPlayer->aiPush(AiQueue::aiContinueRoutine());
```

Comment:
`aiPush` is deferred command, better to refactor this into immediate call (to avoid `clearAiQueue` followup)